srcpkgs/plover/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'plover'
+pkgname=plover
+version=4.0.0.dev12
```

Comment:
This looks like a development release, upstream actually recommends using it instead of v3:

> Note: It is currently recommended to download the unreleased Plover version 4 even if it is not considered "stable" yet. Version 3 is outdated and may not work on some operating systems and contains known bugs.

(From https://github.com/openstenoproject/plover/wiki/Installation-Guide#download-plover)